src/main/java/com/maplexpbar/MapleXPBarPlugin.java

```diff
@@ -23,13 +25,15 @@
 
 
 import net.runelite.client.ui.overlay.OverlayPosition;
+import org.pf4j.Extension;
```

Comment:
As far as I can tell, the main Runelite repository doesn't already use pf4j as a [dependency](https://github.com/runelite/runelite/blob/master/pom.xml), so I'm afraid it will be rejected if this is included. It has been some time since I've worked with runelite plugins, so correct me if I'm wrong. What purpose does the @Extension annotation serve?